about.html

```diff
@@ -0,0 +1,115 @@
+            <li class="nav-link"><a href="#" class="nav_link">Sponsor</a></li>
+            <li class="nav-link"><a href="#" class="nav_link">News</a></li>
+        </ul>
+        <li class="cc-compaign"><a href="#about" class="cc_link">CC compain</a></li>
+    </div>
+
+
+
+        <script src="Javascript.js"></script>
+    </nav>
+    <main>
+        <div class="introduction">
+        <h2 class="topic2">Sports Festival by SportsGala, 2022 - Let's play</h2>
+        <p class="main-paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia ipsum aut distinctio aspernatur exercitationem? Cum quis facere natus accusantium perspiciatis, fugit at quasi in excepturi officia eaque distinctio nemo beatae? Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vero at dolorum neque iusto sit deserunt excepturi repellat porro dolor omnis quas, tempora magni nam. Unde eligendi consequuntur placeat totam ipsa?</p>
+        <div class="footer">
+        <p class="date">Please contact us per Email for nay further questions about Festival! <p>
+           <p class="address"> Kamrankhan.ahmad2@yahoo.com</p>
+        </div>
+    </div>  
+    </main>
+    </header>
+    
+    <div class="background-img3">
+        <div class="main-logo">    
+            <h1 class="logo-header">Sports Festival 2022 logo</h1>
+            <img class="bottom-line2" src="images/Underline.png" alt="underline image">
+            </div>
+                <div class="logo-Description">
+                    <p class="description">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Repellat necessitatibus autem quidem maiores quaerat ab corrupti error. Deserunt necessitatibus reprehenderit doloribus! Assumenda debitis nulla illo deserunt ad cum dolorem? Suscipit. Lorem ipsum dolor sit amet consectetur adipisicing elit.  </p>
+                    <img class="Des-logo" src="images/logo.png">
+                </div>               
+            </div>
+              
+            <div class="background-img3" id="background-img10">
+                <div class="main-logo" id="main-logo">    
+                    <h1 class="logo-header" id="logo-header">See the past Gala sports Festival</h1>
+                    <img class="bottom-line5" id="line2" src="images/Underline.png" alt="underline image">
+                    </div>
+                        <div class="logo-Description">
+                            <p class="description description2">Take a look at the past two Gala sports Festival which took place in Nangarhar and in Herat </p>
+                            <div class="pair"> 
+                                                       
+                            <img class="Des-logo" src="images/cricket-img.jpg">
+                            <img class="Des-logo" src="images/Football-img.jpg">
+                        </div>
+ 
+                        </div>               
+                    </div>
+           
```

Comment:
- [ ] Please use the semantic tag `<section>` to wrap all your individual components instead of the generic tag `<div>` This helps for accessibility and SEO. Please apply this in all your HTML files
- [ ] Please add the orange overlay in the two images contained here to match the design 😃 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/205446247-a99334a9-26e0-4709-9b1b-b6c91407a3f9.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/205446273-a19c2865-ee40-4b82-80fc-ee13c556fb80.png)